app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+# Create Post model
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :likes, class_name: 'Like', foreign_key: :post_id
+  has_many :comments, class_name: 'Comment', foreign_key: :post_id
+
+  after_save :update_posts_counter
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  validates :title, length: { maximum: 250 }
```

Comment:
- [ ] Based on the requirements of this project, the title should not be blank
- Hints `validates :title, presence: true`

Let's kindly fix this issue

Reply:
Fixed